microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/OpenApiManagementClientHelper.java

```diff
@@ -24,9 +24,10 @@
      * @param client {@link OnlineManagementClient} instance used to execute the command
      * @return {@link ModelNodeResult} instance to represent the command execution outcome
      * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
-     *                                          {@link OnlineManagementClient} API
+     *         {@link OnlineManagementClient} API
```

Comment:
Looks like that formatter-plugin does not make this right but I'll not block this PR becuase of it. We can solve it later.